test/data/css/tests.js

```diff
@@ -209,6 +210,25 @@ exports.test_data = {
             }
         ]
     }, {
+        name: "Preserve Newlines",
+        description: "",
+        matrix: [{
+            options: [
+                { name: "preserve_newlines", value: "true" }
+            ],
+            separator_input: '\\n\\n',
```

Comment:
Add at least two more lines to the matrix: one with no newlines and another with more than two newlines (five for example).